boards/px4/fmu-v5x/init/rc.board_sensors

```diff
@@ -2,7 +2,13 @@
 #
 # PX4 FMUv5X specific board sensors init
 #------------------------------------------------------------------------------
-board_adc start
+
+if param compare -s ADC_ADS1115_EN 1
```

Comment:
The ads1115 start via parameter could be done centrally in rc.sensors so other people could potentially use it.

Reply:
So you want to move out the `board_adc start` as well?
I was considering handling multiple adc reports and then extend the battery source param for user selection.

Reply:
Sounds like we can come back to this later for a more unified user configurable ADC solution.

I had some other analog things in mind I was thinking of exposing as standalone fully configurable drivers/modules like analog airspeed, angle of attack, etc.